odbc/src/api/get_diag_field_tests.rs

```diff
@@ -203,7 +199,11 @@ mod unit {
             );
             assert_eq!(
                 "[MongoDB][API] The feature SQLDriv✐𑜲 is not implemented\0",
```

Comment:
Unicode-ASCII characters are... interesting when comparing utf16 and utf32. We end up with an extra trailing 0 in utf32

Reply:
I'm not sure how that's different than what I said 😅 I think maybe what I said just wasn't clear :)